source/_docs/scripts.markdown

```diff
@@ -186,6 +186,7 @@ Delays are useful for temporarily suspending your script and start it at a later
 ```yaml
 # Supports milliseconds, seconds, minutes, hours, days
 # Can be used in combination, at least one required
+# When using milliseconds, consider that delay as *at least* X milliseconds. It won´t be exact.
```

Comment:
With that in mind, this applies to everything in HA basically?
Home Assistant is an async system.

Reply:
Since milliseconds are offered as a valid option for delays, this also creates an expectation with the user; I was expecting I could control the delay up to the millisecond. It would be nice to warn users about this, particularly in this context.

Reply:
Right, the point being with my comment, is that this is not just related to milliseconds 🤷